src/Components/CookieConsentBanner/CookieConsentBanner.jsx

```diff
@@ -0,0 +1,40 @@
+import React from "react";
+import CookieConsent from "react-cookie-consent";
+import { useTranslation } from "react-i18next";
+
+import { pages } from "../../utils/constants.js";
+
+import "./CookieConsentBanner.scss";
+
+import useLinkCreator from "../../hooks/useLinkCreator";
+
+const CookieConsentBanner = () => {
+  const { t } = useTranslation();
+
+  const { isValid: isPrivacyPolicyValid, url: privacyPolicyUrl } =
+    useLinkCreator({ path: pages.privacyPolicy });
+
+  return (
+    <>
+      {isPrivacyPolicyValid && window.VOCASCAN_CONFIG.ENV === "web" && (
```

Comment:
what about adding a dedicated show cookie banner to give the administrator the posibility to disable that feature if the server is only hosted in a local network where its no need for a cookie banner?

Reply:
This is not needed, as the cookie banner is not displayed when no privacy policy is defined. When someone hosts everything for himself, he won't add a privacy policy and won't get the cookie banner displayed.